prow/clonerefs/run.go

```diff
@@ -83,6 +84,16 @@ func (o *Options) createRecords() []clone.Record {
 		oauthToken = token
 	}
 
+	// Print md5 sum of cookiefile for debugging purpose
+	if len(o.CookiePath) > 0 {
+		l := logrus.WithField("http-cookiefile", o.CookiePath)
+		f, err := ioutil.ReadFile(o.CookiePath)
+		if err != nil {
+			l.WithError(err).Warn("Cannot read http cookiefile")
+		} else {
+			l.WithField("md5sum", fmt.Sprintf("%x", md5.Sum(f))).Debug("Http cookiefile md5 sum")
```

Comment:
Info() seems more appropriate here.

Reply:
Already merged. I can create another PR updating this if still desired

Reply:
No need,LGTM!